common/src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/common/model/CurrentIncentive.kt

```diff
@@ -16,7 +18,9 @@ data class CurrentIncentive(
   @Field(type = FieldType.Date, format = [DateFormat.date])
   @Schema(required = true, description = "Schedule new review date", example = "2022-11-10")
   val nextReviewDate: LocalDate? = null,
-)
+) : Diffable<CurrentIncentive> {
+  override fun diff(other: CurrentIncentive): DiffResult<CurrentIncentive> = getDiffResult(this, other)
+}
```

Comment:
Its debatable what diff reporting functionality would be needed per domain, but assuming here for POC that we want everything